server/controllers/api/getlist.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function(app) {
+	
+	var bodyParser = require('body-parser')
+	var jsonParser = bodyParser.json()
+	var urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+	app.use(bodyParser.urlencoded({ extended: false }))
+	app.use(bodyParser.json())
+	
+	app.post('/api/getlist', function (req, res) {
+		console.log('inside post');
+  		if (!req.body) return res.sendStatus(400);
+
+  		var requested = req.body;
+
+  		res.header("Access-Control-Allow-Origin", "*");
+
+		app.db.select([
```

Comment:
The picture data itself will be served statically from our server (normal HTTP behavior).

If the list trees end-point returns all the data of all the trees (including description, type, and image URL) then there is no need for the get detail API end-point.